crates/wasmtime/src/instance.rs

```diff
@@ -204,17 +225,17 @@ impl Instance {
     /// the value, if found.
     ///
     /// Returns `None` if there was no export named `name`.
-    pub fn get_export(&self, name: &str) -> Option<Extern> {
-        let export = self.items.get(name)?;
-        Some(unsafe { Extern::from_wasmtime_export(export, &self.store) })
+    pub fn get_export(&self, store: impl AsContextMut, name: &str) -> Option<Extern> {
```

Comment:
Note that this requires `Mut` where it doesn't actually need it, but I left it here for the same reason as `exports` above.